### PR TITLE
Create Stat Tracker class method from_csv

### PR DESCRIPTION
**Pull request by Joshua-Tukman**
We created the StatTracker method from_csv. We are going to revise when dependent classes have been created.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    @games_data = games_data
+    @teams_data = teams_data
+    @game_teams_data = game_teams_data
+  end
+
+  def self.from_csv(stats_params)
+    games_data = []
+    CSV.foreach(stats_params[:games], headers: true, header_converters: :symbol) do |row|
+      games_params = row.to_hash
+      games_data << games_params
```
Comment (tylerpporter) [inline lib/stat_tracker.rb:21]: Once the Game class has been created, this line will need to be changed to 
`games_data << Game.new(games_params)`

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    end
+
+    teams_data = []
+    CSV.foreach(stats_params[:teams], headers: true, header_converters: :symbol) do |row|
+      teams_params = row.to_hash
+      teams_data << teams_params
```
Comment (tylerpporter) [inline lib/stat_tracker.rb:27]: Once the Team class has been created, this line will need to be changed to 
`teams_data << Team.new(teams_params)`

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    end
+
+    game_teams_data = []
+    CSV.foreach(stats_params[:game_teams], headers: true, header_converters: :symbol) do |row|
+      game_teams_params = row.to_hash
+      game_teams_data << game_teams_params
```
Comment (tylerpporter) [inline lib/stat_tracker.rb:33]: Once the GameTeams class has been created, this line will need to be changed to 
`games_teams_data << GameTeams.new(game_teams_params)`

File: lib/stat_tracker.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+require 'CSV'
+# require './lib/game.rb'
+# require './lib/team.rb'
+# require './lib/game_teams.rb'
```
Comment (tylerpporter) [inline lib/stat_tracker.rb:4]: Once these files and their corresponding classes have been created, these lines will need to be commented back in.

File: lib/stat_tracker.rb [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+class StatTracker
```
Comment (reid-andrew) [inline lib/stat_tracker.rb:6]: This looks really great guys. Nice tight methods, looks to cover all the req's for Iteration 1.

File: lib/stat_tracker.rb (hunk 1 above)
Comment (reid-andrew) [reply lib/stat_tracker.rb:21]: Game class set up in alternate branch to accept params hash.

File: lib/stat_tracker.rb (hunk 3 above)
Comment (reid-andrew) [reply lib/stat_tracker.rb:33]: GameTeams class has been created in alternate branch to accept params hash.

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+coverage
```
Comment (reid-andrew) [inline .gitignore:1]: Note to review git ignore with broader team so we're all on same page as to what this functionality is doing.

File: lib/stat_tracker.rb (hunk 2 above)
Comment (reid-andrew) [reply lib/stat_tracker.rb:27]: Team class created in alt branch to accept params hash as argument.

**Pull request merged**